io_ogre/ui/export.py

```diff
@@ -131,31 +149,65 @@ def execute(self, context):
         if self.filepath == "" or not self.filepath:
             self.filepath = "blender2ogre"
 
-        logger.info("self.filepath %s"%self.filepath)
+        logger.debug("Self.filepath: %s" % self.filepath)
 
         kw = {}
         for name in dir(_OgreCommonExport_):
-            if name[:6] in ('EX_V1_', 'EX_V2_', 'EX_Vx_'):
+            if name.startswith('EX_V1_'):
+                kw[ name[6:] ] = getattr(self,name)
+            elif name.startswith('EX_V2_'):
+                kw[ name[6:] ] = getattr(self,name)
+            elif name.startswith('EX_Vx_'):
                 kw[ name[6:] ] = getattr(self,name)
             elif name.startswith('EX_'):
                 kw[ name[3:] ] = getattr(self,name)
         config.update(**kw)
 
-        print ("_"*80)
+        print ("_" * 80)
+        
         target_path, target_file_name = os.path.split(os.path.abspath(self.filepath))
         target_file_name = clean_object_name(target_file_name)
         target_file_name_no_ext = os.path.splitext(target_file_name)[0]
 
-        logger.info("target_path %s"%target_path)
-        logger.info("target_file_name %s"%target_file_name)
-        logger.info("target_file_name_no_ext %s"%target_file_name_no_ext)
+        file_handler = None
+        
+        # Add a file handler to all Logger instances
+        if config.get('EXPORT_ENABLE_LOGGING') == True:
+            log_file = ("%s\\blender2ogre.log" % target_path)
```

Comment:
```suggestion
            log_file = ("%s/blender2ogre.log" % target_path)
```

please always use forward slashes for paths

Reply:
ok